readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -240,7 +240,7 @@ def build(self):
         if self._force:
             build_command.append('-E')
         if self.config.sphinx.fail_on_warning:
-            build_command.append('-W')
+            build_command.extend(['-W', '--keep-going'])
```

Comment:
Do you think it's better to add them together here? Or it worth to have another config in our config file for the `--keep-going`?

Reply:
I don't see a reason where users don't want these options together